Update espnow documentation (#5207)
OttoWinter/esphomedocs: components/espnow.rst

Automations:

- **on_receive** (*Optional*, :ref:`Automation <automation>`): An automation to perform when data is received. See :ref:`espnow-on_receive`.
- **on_unknown_peer** (*Optional*, :ref:`Automation <automation>`): An automation to perform when data is received from an unknown peer. See :ref:`espnow-on_unknown_peer`.
- **on_broadcast** (*Optional*, :ref:`Automation <automation>`): An automation to perform when a broadcast packet is received.
  See :ref:`espnow-on_broadcast`.

Actions:

- **espnow.sendd**, send a packet to an other device. See :ref:`espnow-send-action`
- **espnow.broadcast**, send a packet to all device within its surroundings. Use this action with care, you can corrupt other device that you maybe not aware of. See :ref:`espnow-broadcast-action`
  
- **espnow.peer.add**, See :ref:`espnow-peer_add-action`
- **espnow.peer.del**, See :ref:`espnow-peer_delete-action`
  
- **espnow.set_channel** with this action you can switch the channel where the packets are being send and received over. **This will only work when *WIFI* is not used**. See :ref:`espnow-set_channel-action`

> 

Reviewer: These are not configuration variables and have entire sections each. Only triggers need to go here
```suggestion
```

- **address** (**Required**, MAC Address): The Peer address that needs to be removed from the list of allowed peers.

.. _espnow-set_channel-action:

``espnow.set_channel`` Action
*****************************

> This is an :ref:`Action <config-action>` to change the channel where espnow is working from.

Reviewer: ```suggestion
This is an :ref:`Action <config-action>` to change the channel that espnow is sending and receiving on.
```

.. code-block:: yaml

    on_...:
      - espnow.set_channel:
          channel: 1
      - espnow.set_channel: 1

Configuration variables:

> - **channel** (**Required**, int): This can be a value between 0 and 15(*). The max number is depending on the country where you using the device. The 0 means that espnow will set the channel number it self (most of the time it would be 1).

Reviewer: ```suggestion
- **channel** (**Required**, int): This can be a value between ``0`` and ``15``. The max number depends on the country where you are using the device. ``0`` means that espnow will set the channel number itself (most of the time it would be ``1``).
```
